src/main/resources/chains/private.conf

```diff
@@ -1,8 +1,20 @@
 {
-  include "base.conf"
-  chain-id = "0x2A"
-  network-id = 42
-  custom-genesis-file = "chains/private-genesis.json"
-  dao = null
-  bootstrap-nodes = []
+    include "base.conf"
+    frontier-block-number = "0"
```

Comment:
I would double check if we do not have some boolean conflicts when are block numbers are equal to 0 (like in ets test sometimes we have fork block equal to `-1` to keep the ordering between forks)